primefaces-cli/src/main/java/org/primefaces/cli/migration/primeflex/PrimeFlexMigration.java

```diff
@@ -57,8 +57,9 @@ protected void initReplaceRegEx() {
         replaceRegex.put("p-col-([0-9]+?)", "col-$1");
         replaceRegex.put("p-col-fixed", "col-fixed");
         replaceRegex.put("p-col-offset-([0-9]+?)", "col-offset-$1");
+        replaceRegex.put("p-offset-([0-9]+?)", "col-offset-$1");
         replaceRegex.put("p-(xl|lg|md|sm)-offset-([0-9]+?)", "$1:col-offset-$2");
-        replaceRegex.put("p-col", "col");
+        replaceRegex.put("p-col(\s|\")", "col$1");
```

Comment:
What about `class='p-col'`?

Reply:
Good catch fixed single quote as well and added test case